Formula/swiftlint.rb

```diff
@@ -19,7 +19,10 @@ def install
   end
 
   test do
-    (testpath/"Test.swift").write "import Foundation\n"
-    system "#{bin}/swiftlint"
+    (testpath/"Test.swift").write "import Foundation"
+    assert_match shell_output("SWIFTLINT_SWIFT_VERSION=3 #{bin}/swiftlint").chomp,
```

Comment:
you have these inverted. It's supposed to be expected, actual not actual, expected.